frontend/src/buttons/ConnectButton/ConnectButton.tsx

```diff
@@ -34,6 +37,13 @@ export const ConnectButton: React.FC<ConnectButtonProps> = ({
     }
   }
 
+  useEffect(() => {
+    if (autoStart && service) {
+      setAutoStart(false)
+      clickHandler()
+    }
+  })
+
```

Comment:
Supports the deep link auto-connect. Didn't work consistently without using state.